concrete/css/build/core/app/components.less

```diff
@@ -73,6 +73,25 @@ ul.item-select-list > li > div.item-select-list-content {
   overflow: hidden;
 }
 
+[data-section=customize-results] {
+  ul.item-select-list {
+     li {
+        padding: 8px;
+        margin-bottom: 2px;
+
+      &:hover {
+          background-color: #e7e7e7;
+          border-radius: 4px;
+          transition: .1s linear background-color;
```

Comment:
@MrKarlDilkington I think that transition should be written like so `transition: background-color .1s linear`